docs/source/tutorial/commercial_paper.md

```diff
@@ -450,135 +357,167 @@ Feel free to examine other files in the `contract` directory to understand how
-command:
+The DigiBank administrator uses the `peer lifecycle chaincode commit` command
+to commit the chaincode definition of `papercontract` to `mychannel`:
+```
+(magnetocorp admin)$ peer lifecycle chaincode commit -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --peerAddresses localhost:7051 --tlsRootCertFiles ${PEER0_ORG1_CA} --peerAddresses localhost:9051 --tlsRootCertFiles ${PEER0_ORG2_CA} --channelID mychannel --name papercontract -v 0 --sequence 1 --tls --cafile $ORDERER_CA --waitForEvent
+```
+The chaincode container will start after the chaincode definition has been
+committed to the channel. You can use the the `docker ps` command to see
+`papercontract` container starting on both peers.
 
 ```
 (magnetocorp admin)$ docker ps
 
-CONTAINER ID        IMAGE                                              COMMAND                  CREATED             STATUS              PORTS          NAMES
-4fac1b91bfda        dev-peer0.org1.example.com-papercontract-0-d96...  "/bin/sh -c 'cd /usr…"   2 minutes ago       Up 2 minutes                       dev-peer0.org1.example.com-papercontract-0
+CONTAINER ID        IMAGE                                                                                                                                                               COMMAND                  CREATED             STATUS              PORTS                                        NAMES
+d4ba9dc9c55f        dev-peer0.org1.example.com-cp_0-ebef35e7f1f25eea1dcc6fcad5019477cd7f434c6a5dcaf4e81744e282903535-05cf67c20543ee1c24cf7dfe74abce99785374db15b3bc1de2da372700c25608   "docker-entrypoint.s…"   30 seconds ago      Up 28 seconds                                                    dev-peer0.org1.example.com-cp_0-ebef35e7f1f25eea1dcc6fcad5019477cd7f434c6a5dcaf4e81744e282903535
+a944c0f8b6d6        dev-peer0.org2.example.com-cp_0-1487670371e56d107b5e980ce7f66172c89251ab21d484c7f988c02912ddeaec-1a147b6fd2a8bd2ae12db824fad8d08a811c30cc70bc5b6bc49a2cbebc2e71ee   "docker-entrypoint.s…"   31 seconds ago      Up 28 seconds                                                    dev-peer0.org2.example.com-cp_0-1487670371e56d107b5e980ce7f66172c89251ab21d484c7f988c02912ddeaec
 ```
 
-Notice that the container is named
-`dev-peer0.org1.example.com-papercontract-0-d96...` to indicate which peer
-started it, and the fact that it's running `papercontract` version `0`.
+Notice that the containers are named to indicate the peer that started it, and
+the fact that it's running `papercontract` version `0`.
 
-Now that we've got a basic PaperNet up and running, and `papercontract`
-installed and instantiated, let's turn our attention to the MagnetoCorp
-application which issues a commercial paper.
+Now that we have deployed the `papercontract` chaincode to the channel, we can
+use the MagnetoCorp application to issue the commercial paper. Let's take a
+moment to examine the application structure.
 
 ## Application structure
 
```

Comment:
It appears that the application structure has changed as well. So we will need to update those.
Things like:
const { FileSystemWallet, Gateway } = require('fabric-network'); should now be 
const { Wallets, Gateway } = require('fabric-network');
and 
const wallet = new FileSystemWallet('../identity/user/isabella/wallet'); should now be
const wallet = await Wallets.newFileSystemWallet('../identity/user/isabella/wallet');
(We should review all them in this section).


These are also under the Application Dependencies section too.

Reply:
Also I think the magnetocorp/application/package.json dependencies might need to be updated too since they still refer to the beta?
```dependencies": {
    "fabric-network": "beta",
    "fabric-client": "beta",
    "js-yaml": "^3.12.0"
  }```